src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
+
+  for (const style of stylesArray) {
+    const [property, value] = style.split(':').map((el) => el.trim());
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
```

Comment:
Instead of using a loop to process each style, consider using the `reduce` method to build the `stylesObject`. This will allow you to avoid the explicit loop and make your code more declarative.